agents/lakeshore372/LS372_agent.py

```diff
@@ -753,10 +784,12 @@ def make_parser(parser=None):
                               second if it is set longer than a channel's dwell\
                               time. This ensures at least one second of data\
                               collection at the end of a scan.")
-    pgroup.add_argument('--auto-acquire', type=bool, default=True,
+    pgroup.add_argument('--auto-acquire', action='store_true',
```

Comment:
This is definitely needed, but we should also update the docs page ocs-site-config example block: `['--auto-acquire', True]` -> `['--auto-acquire'],`

Reply:
Do you know if anybody is currently using this arg? If so this will probably break their setup.

Reply:
Probably everyone. The way it's written right now, does passing `False` even work? Or does site_config handle the type conversion of that bool properly? Unmodified argparse will resolve that to `True` anyway.

If site_config handles it, then I'm tempted to say keep the current implementation so that we don't break configs.